src/doc/book/structs.md

```diff
@@ -88,6 +88,35 @@ fn main() {
 }
 ```
 
+Of course, your structure can still contain `&mut` pointers, which will let
```

Comment:
A comma after "course" does not look right. Either remove it or re-do it like "Your structure can, of course, still...".